commands/cosmetics.py

```diff
@@ -35,14 +35,14 @@ async def cosmetics(self, ctx, operand = None, cosmetic = None):
 
         elif operand.upper() == "SHOW" or operand.upper() == "DISPLAY":
             index = int(cosmetic)
-            inventory = eval(sql.db_query("ibm.db", "SELECT cosmetics FROM Members WHERE UserID = %s" % (str(ctx.author.id)))[0][0])
+            inventory = eval(sql.db_query("SELECT cosmetics FROM Members WHERE UserID = %s" % (str(ctx.author.id)))[0][0])

```

Comment:
Same eval replacement here

Reply:
This is out of scope for this PR. Please raise an issue instead.